src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1117,6 +1117,7 @@ protected override void GetItem(string path)
 
             try
             {
+#if !UNIX
                 bool retrieveStreams = false;
                 FileSystemProviderGetItemDynamicParameters dynamicParameters = null;
```

Comment:
With the current changes, you don't get any error using `copy-item` going from Linux->Windows nor Windows->Linux

I created a new issue https://github.com/PowerShell/PowerShell/issues/4633 to clean up the code later since filesystemprovider.cs already has 21 `#if` statements before I made any changes